src/app/public/modules/dropdown/dropdown-item.component.ts

```diff
@@ -22,6 +22,9 @@ export class SkyDropdownItemComponent implements AfterViewInit {
    * dropdown menu item represents on the web page. For information about ARIA roles, see the
    * [WAI-ARIA roles model](https://www.w3.org/WAI/PF/aria/roles).
    * @default "menuitem"
+   * @deprecated This property should be replaced by placing the role on the interactive
```

Comment:
Looking in the PR preview build, I don't see any updates to the docs. It looks like some of the checks aren't passing, so I pulled down the branch and tried to serve locally, but it failed. So I ran `skyux install` on the branch locally and served again. This time it served, but I still don't see any difference between the live docs and what this branch creates. I don't see the deprecated icon or the text you've added here. Any idea what's going on with this @Blackbaud-TrevorBurch?

Reply:
@Blackbaud-TrevorBurch What's triggering this change or what is it trying to achieve? Why deprecate a menu items role? I'm not familiar with it so not sure what I'm looking for. Thanks.

One thing I did notice is that in the visual test of the dropdown, the programmatic name of the dropdown button is incorrect. It has an aria-label set to "Context menu" when in fact the visual label is "Show dropdown" which should also be the programmatic name. aria-label is higher in the precedence order so it being used for the name.